Pushing vra content fails when blueprint already exist on target vra

<!-- Please search existing issues to avoid creating duplicates. -->

### Description
Pushing vra content fails when blueprint already exist on target vra
Executing task: mvn clean package vrealize:push -DincludeDependencies=false -DskipTests -Pri -pl vra 

### Steps to Reproduce

1. Push vra content containing a blueprint
2. Try to push again 

**Preconditions:** Blueprint exist on target environment

**Expected behavior:** Success release of blueprint with changes

**Actual behavior:** Fails to unreleased existing blueprint version

**Reproduces how often:** Always

**Component/s:** "maven/plugins/vra-ng"]

**Affects Build/s:** 3.1.1

### Environment

#### Client

- Build Tools for VMware Aria Version:3.1.1
- Visual Studio Code Version:1.85.2 
- OS Version:Mac OS

#### Dependencies
Starting VMware Aria Build Tools Checks...
✘ Node.js version 18.20.3 is outside the required range (14 - 16).
✘ Maven version 3.8.8 is older than the required version (>= 3.9).
✔ Java version 17.0.13 is within the required range (17 - 21).
Some checks failed. Please review the above messages.
#### Server

- vRealize Automation Version:8.16
- vRealize Orchestrator Version:8.16

### Failure Logs
ERROR] Failed to execute goal com.vmware.pscoe.maven.plugins:vrealize-package-maven-plugin:3.1.0:push (default-cli) on project vra: Execution default-cli of goal com.vmware.pscoe.maven.plugins:vrealize-package-maven-plugin:3.1.0:push failed: Error ocurred during when unreleasing version 2024-12-06-18-34-42 for blueprint b355c354-a4b6-4d55-800a-8b4232f29b83. Message: 500 Invalid/Unreachable FQDN or IP address: 500 Internal Server Error
[ERROR] Content-Type: application/json
[ERROR] Date: Mon, 09 Dec 2024 10:38:38 GMT
[ERROR] Vary: accept-encoding
[ERROR] Transfer-Encoding: chunked
[ERROR] 
[ERROR] {"timestamp":"2024-12-09T10:38:38.573+00:00","status":500,"error":"Internal Server Error","message":"The request was rejected because the URL contained a potentially malicious String \"//\"","path":"/blueprint/api/blueprints/b355c354-a4b6-4d55-800a-8b4232f29b83/versions/2024-12-06-18-34-42//actions/unrelease"}
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
2024-12-09T10:38:38.053761Z pool-1-thread-1 INFO Stopping configuration XmlConfiguration[location=jar:file:/Users/oviera/.m2/repository/com/vmware/pscoe/iac/artifact-manager/3.1.0/artifact-manager-3.1.0.jar!/log4j2.xml, lastModified=2024-12-09T07:00:12.556Z]...
2024-12-09T10:38:38.053948Z pool-1-thread-1 INFO Configuration XmlConfiguration[location=jar:file:/Users/oviera/.m2/repository/com/vmware/pscoe/iac/artifact-manager/3.1.0/artifact-manager-3.1.0.jar!/log4j2.xml, lastModified=2024-12-09T07:00:12.556Z] stopped.

--- a/common/artifact-manager/src/main/java/com/vmware/pscoe/iac/artifact/rest/RestClientVraNgPrimitive.java
+++ b/common/artifact-manager/src/main/java/com/vmware/pscoe/iac/artifact/rest/RestClientVraNgPrimitive.java
@@ -696,7 +696,7 @@ public class RestClientVraNgPrimitive extends RestClient {
 	public void unreleaseBlueprintVersionPrimitive(final String blueprintId, final String versionId)
 			throws URISyntaxException {
 		URI url = getURI(getURIBuilder().setPath(SERVICE_BLUEPRINT + "/" + blueprintId + SERVICE_BLUEPRINT_VERSIONS
-				+ "/" + versionId + "/" + SERVICE_BLUEPRINT_UNRELEASE_VERSIONS_ACTION));
+				+ "/" + versionId + SERVICE_BLUEPRINT_UNRELEASE_VERSIONS_ACTION));
 
 		try {
 			this.postJsonPrimitive(url, HttpMethod.POST, "");

--- a/docs/versions/latest/Release.md
+++ b/docs/versions/latest/Release.md
@@ -64,6 +64,16 @@ The cheatsheet documents contained similar error:
 
 The cheatsheet documents are properly generated.
 
+### *Unreleasing BP versions would cause an exception*
+
+An exception was thrown like: `The request was rejected because the URL contained a potentially malicious String "//"","path":"/blueprint/api/blueprints/b355c354-a4b6-4d55-800a-8b4232f29b83/versions/2024-12-06-18-34-42//actions/unrelease`
+
+This was because of the `//` after the version id.
+
+#### New Behavior
+
+The extra `/` is removed when forming the url now
+
 ## Upgrade procedure
 
 [//]: # (Explain in details if something needs to be done)

